### DISCUSSION TITLE
Contributing

### DISCUSSION THREAD

**Discussion by Habanerio** [simplcommerce/SimplCommerce#1061] [General]
Hello,
Was wondering how to become an actual contributor to this project?
I've created a fork in my own repo, which has been updated to .Net 8. 
Tests have been added (I've separated them into Unit, Integration, and Functional tests), and have found some small (potential) bugs and included fixes for them.

**Comment by Habanerio** [comment]
Just a follow up, I've been adding tests to my fork. 

You can view diffs @ https://github.com/simplcommerce/SimplCommerce/compare/master...Habanerio:SimplCommerce:master

If there's anything you'd like in this project (such as just tests), let me know and I can create PRs.

[the remark this answers is not in the document]
**Reply by softwebnet** [reply]
Can we remove the PoweredBy : SimpleCommerce and Licence Text?

[the remark this answers is not in the document]
**Reply by hishamco** [reply]
I don't think so. Do you have a problem with the PoweredBy?!!
